syntax = "proto3";

package sum;

message SumOperand
{
    float op1 = 1;
    float op2 = 2;
}

message SumResult
{
    float result = 1;
}

service SumServices{
    rpc ComputeSum(SumOperand) returns (SumResult) {}
}